web/views.py

```
@@ -2907,18 +2909,29 @@ def create_forum_category(request):
 
 @login_required
 def edit_topic(request, topic_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `edit_topic`

(ANN201)